lib/app/views/teams/member_submissions.erb

```diff
@@ -25,7 +25,7 @@
   </p>
   <% if submissions.any? %>
     <ul>
-      <% submissions.each do |submission| %>
+      <% submissions.limit(5).each do |submission| %>
```

Comment:
Hm. I don't think there's a default scope that sets the order. We can use the `reversed` scope to enforce this.